### DISCUSSION TITLE
How to disable image loader caching

### DISCUSSION THREAD

**Discussion by RouHim** [emilk/egui#3541] [Q&A]
Hello everyone,
I would like to deactivate the image loader caching. Is there a way to do this or do I have to call "forget_image()" on the context every time?

Background: A new image is rendered every second by a background process. After rendering it should then displayed / updated in the (noninteractive) gui. But since the bytes:// name is always identical and only the data behind it is updated, the image is not updated.

Thanks & regards

**Comment by RouHim** [comment]
Nevermind, I found a good solution for me.

**Reply by eelgaev** [reply]
I could really use that if you can share what you found.

**Reply by RouHim** [reply]
I did the ui cache eviction by my own.
